### ISSUE TITLE
Implement matrix adjugate for matrices bigger than 3x3

### ISSUE THREAD

**Issue by lukapopijac** [lukapopijac/gaussian-mixture-model#4]

**Comment by lukapopijac** [comment]
Maybe not needed after #11 is done. Then direct calculation of matrix inverse is easy, and adjugate in no longer needed.

**Comment by lukapopijac** [comment]
Won't be implemented. 
Not needed as inverse is being directly calculated for matrix bigger than 3x3.
